### DISCUSSION TITLE
Great video about what file coin is and how it works

### DISCUSSION THREAD

**Discussion by murrlincoln** [filecoin-project/community#246] [News and Events]
What is Filecoin? Air BnB for your computer? Coin Talk with Ben ep.5

https://www.youtube.com/watch?v=TQH0V5Zx74U

**Comment by klauskobald** [comment]
yes, but how do I actually do it?

How do I earn money with sharing storage? I am confused!
There is filecoin, ipfs, lotus, mining. I installed the Desktop app and understand I can share my imported files to anywhere. Great - that works.
I have 20TB of free diskspace. How do I share that and get filecoins for it? I read that a raspberry pi is enough.Then I read i need 8 core 128GB RAM. 
Is there a doc that clarifies all these relations?
Thanks!
